inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/project/ProjectRecommendersPanel.html

```diff
@@ -18,11 +18,13 @@
 -->
 <html xmlns:wicket="http://wicket.apache.org">
 <wicket:panel>
-  <div class="flex-content flex-v-container flex-gutter">
-    <div class="flex-content flex-h-container flex-gutter">
-      <div class="flex-sidebar flex-v-container flex-only-internal-gutter" wicket:id="recommenders" style="flex-basis: 30%;"></div>
-      <div class="flex-content flex-v-container flex-only-internal-gutter" wicket:id="recommenderEditor"></div>
-    </div>
-  </div>
+	<div class="flex-content flex-v-container flex-gutter">
```

Comment:
This file contains lots of whitespace changes that need to be reverted. Please note that HTML files are indented using spaces only and using 2 spaces for an indentation.